app/views/home/index.html.erb

```diff
@@ -1,3 +1,13 @@
 <% if current_user %>
-  <p><%= link_to 'My Picks', {controller: :picks, action: :index}, method: :get %></p>
+  <p><%= link_to 'My Picks', {controller: :picks, action: :new}, method: :get %></p>
 <% end %>
+
+<% if controller.action_name == "index" %>
```

Comment:
This should never be false — this view (`home/index.html.erb`) will only ever be used by `HomeController#index`.

Reply:
This was another thing I did just to learn the language. I deleted my chickenscratch